packages/swap/src/translators/reactjs.tsx

```diff
@@ -151,6 +151,27 @@ export const useOrderByStatusOpen = (
   return query.data
 }
 
+export const useCreateOrder = (
+  options?: UseMutationOptions<
+    Swap.CreateOrderResponse,
+    Error,
+    Swap.CreateOrderData
+  >,
+) => {
+  const {order} = useSwap()
+
+  const mutation = useMutationWithInvalidations({
```

Comment:
nitpicky but conceptually if needs the returned data, it is a query not a mutation -> mutations in general should expect only done/error. (don't need to change it)